graphql/schema.graphql

```diff
@@ -119,6 +119,7 @@ input UserSettingsInput {
   githubUser: GithubUserInput
   slackUsername: String
   notifications: NotificationsInput
+  useSpruceOptions: UseSpruceOptionsInput
```

Comment:
What are you planning to use this parameter for? I would think that it's just something that gets returned but I'm probably missing something

Reply:
When the user views the welcome modal we want to toggle this field so we dont show them the modal more than once.  I will also be using this field for the opt in/out ability of spruce.

Reply:
Doh right, wasn't thinking there